src/linuxforhealth/csvtofhir/pipeline/tasks.py

```diff
@@ -13,6 +13,7 @@
 from linuxforhealth.csvtofhir.support import read_csv
 # Do not remove, required for _build_status_history task
 from linuxforhealth.csvtofhir.model.csv.encounter import EncounterStatusHistoryEntry
+from linuxforhealth.csvtofhir.model.csv.basic import TokenDetails
```

Comment:
If this is really needed, let's put a comment like row 14 so it is not deleted accidentially

Reply:
Added "# Do not remove, required for build_object_array task"